workflow/tests/base_results/results_simulations_loads.csv

```diff
@@ -427,7 +427,7 @@ base.xml,22.502,0.0,13.745,9.071,0.615,0.0,0.0,0.0,3.819,3.882,0.545,7.57,0.682,
 house001.xml,19.136,0.0,53.592,10.404,2.718,0.0,0.0,0.515,2.094,7.773,0.453,0.0,1.007,7.209,-5.737,0.0,0.0,0.479,0.897,-0.554,4.463,0.0,5.355,0.0,3.721,-5.804,-2.913,0.595,1.747,4.061,0.325,0.0,0.255,2.786,13.063,0.0,0.0,0.514,6.356,-0.54,-0.263,-2.042,-0.526,0.0,12.326,10.704,4.463
 house002.xml,11.756,0.0,41.956,7.517,2.933,0.0,0.0,0.0,2.717,5.297,0.0,0.0,0.846,5.655,-4.461,0.0,0.0,0.0,1.582,-0.314,1.608,0.0,3.843,0.0,1.537,-4.296,-2.401,0.0,2.777,2.994,0.0,0.0,0.411,1.123,10.014,0.0,0.0,0.0,8.095,-0.308,-0.16,-1.477,-0.583,0.0,6.71,8.58,3.977
 house003.xml,12.794,0.0,44.159,7.517,2.721,0.0,0.0,0.673,3.003,4.939,0.0,0.0,0.981,6.172,-4.47,0.0,0.0,0.0,0.958,-0.33,2.043,0.0,4.013,0.0,1.784,-4.533,-2.624,0.82,2.74,2.797,0.0,0.0,0.642,1.928,11.287,0.0,0.0,0.0,6.2,-0.323,-0.184,-1.547,-0.563,0.0,7.521,8.821,4.248
-house004.xml,36.804,0.0,114.385,8.975,3.543,0.0,0.0,0.127,5.222,11.191,0.0,0.0,1.244,13.586,-5.978,0.0,0.0,0.0,3.21,-0.738,5.025,0.0,6.238,0.0,7.332,-6.14,-3.826,0.199,5.894,11.327,0.0,0.0,0.506,8.875,20.819,0.0,0.0,0.0,18.642,-0.726,1.046,0.0,1.82,0.0,23.987,14.638,7.729
+house004.xml,36.804,0.0,114.385,8.975,3.543,0.0,0.0,0.127,5.226,11.195,0.0,0.0,1.243,13.583,-5.986,0.0,0.0,0.0,3.215,-0.736,5.025,0.0,6.239,0.0,7.333,-6.142,-3.827,0.199,5.897,11.33,0.0,0.0,0.506,8.872,20.811,0.0,0.0,0.0,18.648,-0.724,1.047,0.0,1.821,0.0,23.987,14.636,7.728
```

Comment:
Hmm, why would this change?

Reply:
Can you identify the cause?

Reply:
https://github.com/NREL/OpenStudio-HPXML/pull/1784/commits/33a863cd9cfd1081857613b60b9f09c4930ad34b